src/lib/middleware/morgan.ts

```diff
@@ -26,12 +26,22 @@ function skipAssets(req: ArtsyRequest, res: ArtsyResponse): boolean {
   )
 }
 
+const BANNED_PARAMS = ["setup_intent_client_secret"]
+export function filterParams(url: string): string {
+  const cleaned = BANNED_PARAMS.reduce((prevUrl, key) => {
+    const re = new RegExp(`${key}=([^&]+)`)
+    return prevUrl.replace(re, (match, p1) => match.replace(p1, "[FILTERED]"))
```

Comment:
Can we rename `p1` to something meaningful to someone who has no knowledge of security lingo?